create_MyString_file.py

```diff
@@ -0,0 +1,20 @@
+class MyString:
+    def __init__(self,s):
+        self.string = s
+
+    def len(self):
+        count = 0
+        for char in self.string:
+            count += 1
+        return count
+
+    def add_string(self, new_str):
+        return "{}{}".format(self.string, new_str)
```

Comment:
we don't use `format` method, suggest for loop

Reply:
https://www.programiz.com/python-programming/operator-overloading#:~:text=Python%20operators%20work%20for%20built,lists%2C%20or%20concatenate%20two%20strings.

Reply:
suggest write operator

Reply:
ok. I got it. Thanks